Index: src/test/java/src/api_learning/SwipeHorizontal.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/test/java/src/api_learning/SwipeHorizontal.java b/src/test/java/src/api_learning/SwipeHorizontal.java
--- a/src/test/java/src/api_learning/SwipeHorizontal.java	
+++ b/src/test/java/src/api_learning/SwipeHorizontal.java	
@@ -54,6 +54,8 @@
                         .moveTo(endPoint)
                         .release()
                         .perform();
+
+                MobileElement slideElem = appiumDriver.findElement(MobileBy.xpath(""))
             }
 
         }
